ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml

```diff
@@ -37,6 +37,12 @@ FAN:
   implements:
   - EQUIPMENT
 
+AWSH:
```

Comment:
If we are modeling these at MAUs, then this is unnecessary. Revert.

Reply:
AWSH from general types has been removed...